client/reader/stream/customer-council-banner.jsx

```diff
@@ -0,0 +1,56 @@
+import { Reader, SubscriptionManager } from '@automattic/data-stores';
+import Banner from 'calypso/components/banner';
+import { navigate } from 'calypso/lib/navigate';
+
+export const CustomerCouncilBanner = ( { translate } ) => {
+	const CUSTOMER_COUNCIL_P2_URL = 'https://readercouncilgeneral.wordpress.com/';
+	const CUSTOMER_COUNCIL_P2_ID = '237686330';
```

Comment:
Note - Being constants like this we could move them out of the component (have them live between imports and the banner component) so they can be exported as well. Although, we don't currently have any need for that and can always move these in another diff if we do end up having that need. Anyways, this is totally fine right now. 😄

Reply:
Duh! 🤦‍♂️. I originally had this component inlined in the ReaderStream component file while iterating and defined these in the component to keep things tidy but I overlooked moving them out when I move this to its own file. Moving them will also keep them from being redefined each render too, so thanks for the callout!